principles/3_Privacy.md

```diff
@@ -0,0 +1,51 @@
+# Privacy
+
+## Data Minimization
+
+* The minimum amount of data possible shall be collected, for the purposes for which it is gathered. Data that is not used in the disclosure to healthcare authorities should not be collected unless it is essential to the purposes of the application
+* Users can choose how much data to share from symptom surveys
+
+## Consent
+* A member of the public (MoP) shall consent to data being collected or transmitted explicitly, that is it should not be implied by usage, or mixed with another consent
+* A MoP shall provide informed consent, that is they must be able to understand what they are consenting to and the purposes for which each category of data is gathered
+* Information about privacy shall be communicated in plain English, not legal, or technical language
+* Consent to collect data shall be separate from consent to disclose data, and include retention periods
+* Consent to collect each category of data (e.g. location, proximity) shall be separate
+* No communication initiated by 3rd parties shall be provided to a MoP, through the app, without their consent
+* No data shall be shared with 3rd parties, except healthcare authorities using the agreed mechanisms
+* The user of SafePaths shall be able to disable location tracking temporarily
+* Consent to disclose data to a HA shall be separate, at the time of disclosure, and contain a reference to the HA privacy policy
+
+## Transparency & Accountability
+
+* The privacy policy shall include the steps taken by the data collector to ensure the confidentiality, integrity, and quality of the data
+* The data controller, that is, who controls what data is collected and how it is stored and disclosed shall be identified clearly
+* No single person shall be able to modify the source code and release the application to the public
+* All source code, models and technical documentation shall be publicly available, including all dependencies
+* An assessment of any residual privacy risks shall be made public in the form of a risk assessment
+
+# Anonymization
+
+* Location, proximity, health, race/ethnicity, gender, and any data that allows the identification of a person, shall be considered sensitive unless it is sufficiently aggregated
+* Paths between waypoints shall not be stored or published
+* No identifiers of nearby devices shall be recorded that can be used to identify a MoP
+* Healthcare authorities should have the ability to effectively redact data
+* Un-redacted data shall not be persisted by a Healthcare authority in the Safe Places web application or it’s supporting components
```

Comment:
I'm not sure what this means. All data is deleted after x days? Also, what about future analysis of data? I don't know to what extent we intend to make that possible, but it sounds like this could prevent that.

Reply:
When I read this before, I assumed that 32 and 33 referred to individual data and not to aggregated data, and that individual data is deleted after x days while aggregated data is stored longterm. If this is correct, we could specify which we are referring to for clarity.

Reply:
A MoP might be asked, seperately from SafePaths/SafePlaces if they consent to their data being kept for some other explicit purpose like long term analysis? It is their data after all.
HA's have all kinds of regulations around the handling of health data, they may well need to redact even the aggregated data at times, like if they are asked to remove someones data from the aggregation, or if they determine some error has occurred.

Reply:
Regarding #30 - Seems like this precludes storing information about public transit routes. Do we want to do that (if it does)?

Reply:
> Regarding #30 - Seems like this precludes storing information about public transit routes. Do we want to do that (if it does)?

Amended to " Paths followed by individuals between waypoints shall not be stored or published"

Reply:
> When I read this before, I assumed that 32 and 33 referred to individual data and not to aggregated data, and that individual data is deleted after x days while aggregated data is stored longterm. If this is correct, we could specify which we are referring to for clarity.

I meant both - updated to be clearer

Reply:
> A MoP might be asked, seperately from SafePaths/SafePlaces if they consent to their data being kept for some other explicit purpose like long term analysis? It is their data after all.

I suppose that could happen, but it isn't supported by our technical solution.  I can imagine that if that consent was gained outside of our process, the HA could build a mechanism to extract data...